src/main/java/org/elasticsearch/common/jna/Kernel32Library.java

```diff
@@ -60,6 +83,26 @@ public static Kernel32Library getInstance() {
         return Holder.instance;
     }
 
+    public boolean virtualLock(long address, long size) {
+        return internal.VirtualLock(address, size);
+    }
+
+    public int virtualQueryEx(int handle, Long address, MEMORY_BASIC_INFORMATION memoryInfo, int length) {
```

Comment:
I saw the following advice: 
https://github.com/twall/jna/blob/master/www/FrequentlyAskedQuestions.md#how-do-i-map-a-native-long-type

I'm not sure if it concerns this code but I'm wondering how `long address` behaves on 32 and 64 bit platforms.